File: learning/tyama_paizalearning~fizz-buzz.php
#!/usr/bin/php
<?php
$n=intval(fgets(STDIN));
for($i=1;$i<=$n;$i++){
	echo ($i%15==0?"Fizz Buzz":($i%3==0?"Fizz":($i%5==0?"Buzz":$i))).PHP_EOL;
}

File: learning/tyama_paizalearning~fizz-buzz.c
#include <stdio.h>
int main(){
	int i=1,n;
	scanf("%d",&n);
	for(;i<=n;i++){
		printf(i%15==0?"Fizz Buzz\n":i%3==0?"Fizz\n":i%5==0?"Buzz\n":"%d\n",i);
	}
}
